src/components/molecules/NavBar/NavBar.scss

```diff
@@ -1,6 +1,6 @@
 @import "scss/constants.scss";
 
-$menu-item-size: $spacing--xxl;
+$menu-item-size: $spacing--xxl + 2px;
```

Comment:
This shouldn't be a thing, I mean, isn't `$spacing--xxl` large enough? Does it really need 2 more pixels?

The idea behind those `$spacing--*` constants is to have more unified sizes. Adding few pixels here and there defeats the purpose of them.

Reply:
replaced with `spacing-lg * 2` 4af31d2